profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.install

```diff
@@ -48,3 +50,26 @@ function tmgmt_poetry_mock_uninstall() {
     entity_delete('tmgmt_translator', $translator->tid);
   }
 }
+
+/**
+ * Implements hook_disable().
+ */
+function tmgmt_poetry_mock_disable() {
+  // Cleaning TMGMT jobs and related items.
+  $jobs = PoetryMock::getAllTmgmtJobs();
+  foreach ($jobs as $job) {
+    $job_items = $job->getItems();
+    foreach ($job_items as $item) {
+      $item->delete();
+    }
+    $job->delete();
+  }
+
+  // Deleting files which were created by tmgmt_poetry_mock.
+  PoetryMock::removeAllRequestTranslationFiles();
+  file_unmanaged_delete_recursive(TMGMT_POETRY_MOCK_REQUESTS_PATH);
+
```

Comment:
I don't really agree that, by disabling the module, the poetry tables should be truncated. I see more problems than advantages with this. One problem is someone some day tries this module in production but as this module is for testing proposes and it is hidden, may be it is an acceptable risk (although it would have disastrous impacts).
But even for development proposes... do I really want to truncate the poetry tables if I disable this module? And couldn't it be confusing that this occurs silently, without me asking or being informed? Imagine I got a dump from production and at some point I disable the mock...